package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-prettier",
-  "version": "2.6.2",
+  "version": "2.6.3",
```

Comment:
Sorry, I should have mentioned this before, but it's also not necessary to update the version in `package.json` (it will also be automatically updated on release).